code/command_block/command_block.py

```diff
@@ -0,0 +1,70 @@
+import os
+from time import sleep
+import subprocess
+import configparser
+
+lastLine = None
+sensor_logs = "foo"
+text_buffer = "buffer.txt"
+lora_status_file = "lora_status.txt"
+rx_file = "foo/rx.txt"
+
+def image_request(line):
+    camera_log = open(sensor_logs + "camera.txt")
+    lora_status = open(lora_status_file, "a")
+    tstamps = camera_log.readlines()
+    t = int(line[1:6])
+    q = line[6]
+    ti = min([abs(t-int(x)) for x in tstamps])
+    tim = t - ti if str(t - ti)+"\n" in tstamps else t + ti
+    lora_status.write("I " + str(tim) + " " + str(q) + '\n')
+    lora_status.close()
+
+def thermal_request(line):
+    thermal_log = open(sensor_logs + "thermal.txt")
+    lora_status = open(lora_status_file, "a")
+    tstamps = thermal_log.readlines()
+    t = int(line[1:6])
+    ti = min([abs(t-int(x)) for x in tstamps])
+    tim = t - ti if str(t - ti)+"\n" in tstamps else t + ti
+    lora_status.write("T " + str(tim) + " " + 0 + '\n')
+    lora_status.close()
+
+def system_reboot():
+    cmd = "reboot"
+    print("rebooting")
+    subprocess.Popen(cmd)
+
+def send_system_info():
+    lora_status = open(lora_status_file, "a")
+    lora_status.write("S 0 0")
```

Comment:
What do these zeros mean? Do we gather system info somewhere already?
Lora would also prefer fixed length lines in lora_status.txt

Reply:
These are some kind of foo numbers, so each line in Lora status file has a format: Char Int Int

Reply:
Gonna change it into S00... so commands have equal length

Reply:
current code produces the following output:
```python
>>> "S{:09}\n".format(0)
'S000000000\n'
```
It differs in number of zeros, is that intentional?

Reply:
Already Changed in new commit: df5d08f